docs/versioned/api/java-api.md

```diff
@@ -251,6 +251,8 @@ This project Iceberg also has modules for adding Iceberg support to processing e
 
 * `iceberg-spark2` is an implementation of Spark's Datasource V2 API in 2.4 for Iceberg (use iceberg-spark-runtime for a shaded version)
 * `iceberg-spark3` is an implementation of Spark's Datasource V2 API in 3.0 for Iceberg (use iceberg-spark3-runtime for a shaded version)
+* `iceberg-spark-3.1` is an implementation of Spark's Datasource V2 API in 3.1 for Iceberg (use iceberg-spark-3.1-runtime for a shaded version)
+* `iceberg-spark-3.2` is an implementation of Spark's Datasource V2 API in 3.2 for Iceberg (use iceberg-spark-3.2-runtime for a shaded version)
```

Comment:
I thought these were `iceberg-spark-runtime-3.x`. Did we not make that change?

Reply:
Also, maybe we shouldn't put the information about which jar to use in parentheses? Seems to me like it downplays it and we'll get questions about "why isn't this iceberg-spark-3.1 jar working?". But that's less important than just updating these two to have the correct artifact name 🙂

Reply:
Woops bad copy paste from my cli